opensearch-operator/pkg/builders/cluster_test.go

```diff
@@ -347,4 +355,36 @@ var _ = Describe("Builders", func() {
 			Expect(result).To(BeFalse())
 		})
 	})
+
+	When("Using custom command for OpenSearch startup", func() {
+		It("it should use the specified startup command", func() {
```

Comment:
IMO the test can be simpler. Just do your check on the sts definition returned by `NewSTSForNodePool`. No need to actually create it in kubernetes. Would also make the test faster.

Reply:
Fixed as per comments, please review.